vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -126,12 +135,21 @@ def _optimise_fig_layout_for_dashboard(fig):
                 # Reduce `margin_t` if not explicitly set.
                 fig.update_layout(margin_t=64)
 
-            if fig.layout.title.pad.t is None and "<br>" not in fig.layout.title.text:
```

Comment:
For reviewers: If someone uses the title argument inside a px chart, we've decided to not try and align it with other components anymore. Only if they use the `Graph.title` it will be properly aligned. A warning is raised when they use the `title` argument inside the px.chart